src/ui/pages/ProfilePage.js

```diff
@@ -0,0 +1,37 @@
+import { testStep, expect } from "../../common/helpers/pwHelpers.js";
+
+const PROFILE_UPDATED_TEXT = /Profile Updated/i;
```

Comment:
better to create a separate file like `src`/`common`/`constants` -> `profile.constants.js`